### PR TITLE
Feature/edit stage

### PR DESCRIPTION
**Pull request by wwqrd**
This implements a single stage editor for the "information interface".

Features include:
- Creating a protocol ' project'
  - This should create a '.canvas' folder, with a protocol.json and an empty 'assets/' directory
- Loading that protocol 'project'
  - Should load up the protocol and display the protocol editor
- Saving the loaded protocol project
  - Should update protocol.json in the currently loaded protocol
- Editing an instructions stage, with asset importing.
  - when adding an asset it should copy it to the 'assets/' folder and display a preview.

## Known issues
- UI assets aren't showing up correctly, fonts, and images from public
- There's no way to return to the homepage
- Some visual glitches with the guidance when dragging and dropping items

### CODE REVIEW

File: src/components/StageEditor/sections/ContentItems.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                updateItem={this.updateItem}
+                deleteItem={this.deleteItem}
+                onSortEnd={this.moveItem}
+                lockAxis="y"
+                useDragHandle
+              />
+            }
+
+            <div className="stage-editor-section-content-items__controls">
+              <AddButton onClick={() => this.createNewItem('text')} type="text">Copy</AddButton>
```
[the remark this answers is not in the document]
Comment (wwqrd) [reply src/components/StageEditor/sections/ContentItems.js]: "Copy" the Noun :) I'll rename it.

[the remark this answers is not in the document]
Comment (jthrilly) [reply src/components/StageEditor/sections/ContentItems.js]: You mean text. I get it now. Doh.

**Comment by wwqrd** [comment]
I've changed the file names and 'copy' button.

> - add a 'label' property which is a short description for the stage to be used in the menu.
> - rename contentItems to be items

I've implemented the above in: https://github.com/codaco/Architect/pull/24

**Comment by wwqrd** [comment]
To make tests pass there are some issues with the colorDictionary that I need to look into. Strictly speaking it's out the scope of this task so I'll make a new issue for it.

**Comment by wwqrd** [comment]
For some reason this won't build on travis, but it builds locally. I'm going to go ahead and merge this and revisit that later.

**Pull request merged**